src/parser-browsers.js

```diff
@@ -23,10 +23,7 @@
  * return true/false to get the Parser know if this browser descriptor matches the UA or not.
  */
 
-import {
-  getFirstMatch,
-  getSecondMatch,
-} from './utils';
+import Utils from './utils.js';
```

Comment:
Sorry, but I don't understand, why we need to replace extracted `getFirstMatch` and `getSecondMatch` methods to the `Utils`. I thought all we needed is the ES2015 module export changes in `utils.js` and the importing part would have a `.js` added in the end. Do I missing something?

Reply:
Totally agree this isn't ideal - it happens is because Utils is currently a class with static methods.
With 'pure' es6 you can't import methods separately from a class (I didn't even realise it was possible with webpack until I saw it here :) )

The ideal option - which I mentioned in the original post above  - would be to update the utils.js file into separate exports rather than static methods on a class. I'll update the PR later so you can have a look and see if it's what you want (should be back to looking how it was before except internally utils will export each function rather than export one class)

Reply:
Oh, right. My mistake, sorry. I've forgotten about the class. Actually, having one export produces less code in the end, because after all each module has it's own wrapper when babel transpiles it to ES5. So, no worries then and we can use it like that.